### PR TITLE
Style all /about content

### LINKED ISSUE #395: Style all /about content
- [x]  Once content is added, style these about pages consistently
    - [x]  Shared CSS across all subpages (currently copied over in each)
        - Should prob just add classes & use global css rules vs. scoped files to each component
    - [x]  Add spacing between lists

### PR DESCRIPTION
**Pull request by corwintines**
Fixes: https://github.com/ethereum/clrfund/issues/395

**What Changed**
- Moved about styles into _themes.scss
- Import themes file into /about pages
- Some styling around li

Note: Open to feedback on li styling. Tried to give some more breathing room to make them easier to read.

### CODE REVIEW

File: vue-app/src/views/About.vue [hunk 1]
```diff
@@ -132,25 +132,5 @@ export default class About extends Vue {}
 <style scoped lang="scss">
 /* TODO apply these styles across all About subpages  */
```
Comment (samajammin) [inline vue-app/src/views/About.vue]: Remember to delete these comments 😄  - I'll do that now.

File: vue-app/src/views/About.vue [hunk 2]
```diff
@@ -132,25 +132,5 @@ export default class About extends Vue {}
...
-p {
-  font-size: 16px;
-  line-height: 30px;
-}
-
-li {
-  font-size: 16px;
-  line-height: 150%;
-}
+@import '../styles/theme';
```
Comment (samajammin) [inline vue-app/src/views/About.vue]: I don't actually think this is necessary, since we already import the theme in the App.vue 🧐

Comment (samajammin) [reply vue-app/src/views/About.vue]: I think we can remove these entire style blocks from these components.

**Comment by samajammin** [comment]
FYI @corwintines I removed all the style tags from those components:
```
<style scoped lang="scss">
@import '../styles/vars';
@import '../styles/theme';
</style>
```
From my understanding they aren't needed since we already import in App.vue & that style tag isn't scoped.

**Review by samajammin — APPROVED** [review]
Checked the preview deploy after my commits & the styles are still applied 👍

**Comment by corwintines** [comment]
> FYI @corwintines I removed all the style tags from those components:
> 
> ```
> <style scoped lang="scss">
> @import '../styles/vars';
> @import '../styles/theme';
> </style>
> ```
> 
> From my understanding they aren't needed since we already import in App.vue & that style tag isn't scoped.

Ah cool. Thanks for doing the cleanup on that!

**Pull request merged**